guide/src/commands/upgrade/test.md

```diff
@@ -86,7 +86,7 @@ gaiad version --log_level error --long | head -n4
      > `gm ports` in order to see a list of the ports being used.
 
     ```shell
-    gaiad --node tcp://localhost:<RPC PORT> query gov proposal 1 --home $HOME/.gm/ibc-0/
+    gaiad --node tcp://localhost:27000 query gov proposal 1 --home $HOME/.gm/ibc-0/
```

Comment:
Should the RPC port here be specified? Is it always the case that this is always the correct port?

Reply:
Yes, it's always `27000` since we specify the `gm` config at the beginning of the tutorial